media-libs/embree/embree-3.13.0.ebuild

```diff
@@ -0,0 +1,107 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake flag-o-matic linux-info toolchain-funcs
+
+DESCRIPTION="Collection of high-performance ray tracing kernels"
+HOMEPAGE="https://github.com/embree/embree"
+LICENSE="Apache-2.0"
+KEYWORDS="~amd64 ~x86"
+SRC_URI="https://github.com/embree/embree/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SLOT="3"
```

Comment:
Please keep the above sorted according to the skeleton file with proper spacing between the blocks.